src/service-override/files.ts

```diff
@@ -468,14 +470,14 @@ void userDataFileSystemProvider.mkdir(URI.from({ scheme: Schemas.vscodeUserData,
 
 const providers: Record<string, IFileSystemProvider> = {
   extension: extensionFileSystemProvider,
```

Comment:
As a sidenote, in the future (for the extension-gallery PR) we need to find a new name for this, as `extension` is used by the Extension Gallery to show extension details (with the button to install).

Reply:
Sure, we can change it to whatever we want